api/src/main/kotlin/edu/wgu/osmt/config/AppConfig.kt

```diff
@@ -31,6 +31,12 @@ class AppConfig(
     @Value("\${app.loginSuccessRedirectUrl}")
     val loginSuccessRedirectUrl: String,
 
+    @Value("\${app.userName}")
```

Comment:
I wonder if we should provide a default for these

@Value("\${app.userName:email}")